src/mscorlib/src/System/Runtime/Intrinsics/X86/Sse2.PlatformNotSupported.cs

```diff
@@ -263,19 +263,19 @@ public static class Sse2
         /// <summary>
         /// __m128i _mm_cvtps_epi32 (__m128 a)
         /// </summary>
-        public static Vector128<int> ConvertToInt(Vector128<float> value) { throw new PlatformNotSupportedException(); }
+        public static Vector128<int> ConvertToInt32(Vector128<float> value) { throw new PlatformNotSupportedException(); }
```

Comment:
This should be `ConvertToVector128Int32` to match the naming convention the other methods have been given.